README.md

```diff
@@ -4,42 +4,8 @@
 # OpenUI5 Website
 
 This is the repository for the [openui5.org](openui5.org) website of the [OpenUI5](https://github.com/SAP/openui5) framework.
-We welcome contributions to our website. If you have a proposal you want to share with us, follow the instructions below.
 
-## Contributing
```

Comment:
We can't remove this section as its own according to our guidelines. Please note how it's done in https://github.com/SAP/ui5-tooling and do the similar way.